docs/classic-ui/templates-global-variables.md

```diff
@@ -0,0 +1,58 @@
+---
+html_meta:
+  "description": "Convenience global Plone variables in templates"
+  "property=og:description": "Convenience global Plone variables in templates"
+  "property=og:title": "Global variables in templates"
+  "keywords": "Plone, Classic UI, global variables, templates"
+---
+
+# Global variables in templates
+
+For convenience Plone defines a couple of global variables often used in templates.
+
+- {ref}`backend-global-utils-portal-state-label`
+- {ref}`backend-global-utils-context-state-label`
+- {ref}`backend-global-utils-plone-view-label`
+- {ref}`icons <classic-ui-icons-icon-resolver-label>`
+- `lang`
+- {ref}`classic-ui-templates-global-variables-portal-url-label`
+- {ref}`classic-ui-templates-global-variables-check-permissions`
+- `ajax_include_head`
+- `isAnon`
+
+
+(classic-ui-templates-global-variables-portal-url-label)=
+
+## `portal_url`
+
+`portal_url` will return the portal URL.
+It can be used as follows:
+
+```xml
+<a href="${portal_url}/@@overview-controlpanel>Overview Controlpanel</a>
+```
+
+It is basically a short cut for:
+
+```xml
+<div tal:define="portal_url python: portal_state.portal_url()">
+  <a href="${portal_url}/@@overview-controlpanel">Overview Controlpanel</a>
+</div>
+```
+
+
+(classic-ui-templates-global-variables-check-permissions)=
+
+## `checkPermission`
+
+`checkPermission` will return the checkPermission method from portal_membership tool.
```

Comment:
```suggestion
`checkPermission` will return the `checkPermission` method from `portal_membership` tool.
```

Reply:
And is that `portal_membership` tool code or a UI element? If the latter, then it should use the guilabel or menu(?) directive.

Reply:
it's code, it's a tool available in the root of every Plone site and provides methods to manager users and check permissions and so on.

Reply:
If they are code, then they should be inline literals surrounded by single backticks, as shown in my suggestion. I'm not sure why you resolved this one without accepting the suggestion? Please let me know. Thank you!